docs/tables/yml_key_value.md

```diff
@@ -0,0 +1,237 @@
+  start_line
+from
+  yml_key_value;
+```
+
+```sh
++----------------------+-----------------------------+------------+-----------------------------+------------+
+| key_path             | value                       | value_type | pre_comments                | start_line |
++----------------------+-----------------------------+------------+-----------------------------+------------+
+| items.1.size         | 8                           | !!int      | []                          | 15         |
+| customer.family_name | Gale                        | !!str      | []                          | 6          |
+| items.1.part_no      | E1628                       | !!str      | []                          | 13         |
+| items.0.part_no      | A4786                       | !!str      | ["# List of ordered items"] | 9          |
+| items.0.price        | 1.47                        | !!float    | []                          | 11         |
+| date                 | 2012-08-06                  | !!str      | []                          | 3          |
+| items.1.price        | 133.7                       | !!float    | []                          | 16         |
+| customer.first_name  | Dorothy                     | !!str      | []                          | 5          |
+| includes.0           | common.yaml                 | !!str      | []                          | 3          |
+| foo                  | bar                         | !!str      | []                          | 4          |
+| items.1.description  | High Heeled "Ruby" Slippers | !!str      | []                          | 14         |
+| receipt              | Oz-Ware Purchase Invoice    | !!str      | []                          | 2          |
+| items.0.quantity     | 4                           | !!int      | []                          | 12         |
+| items.0.description  | Water Bucket (Filled)       | !!str      | []                          | 10         |
+| city                 | East Centerville            | !!str      | []                          | 22         |
+| items.1.quantity     | 1                           | !!int      | []                          | 17         |
+| state                | KS                          | !!str      | []                          | 23         |
+| bill_to              | <null>                      | !!null     | []                          | 18         |
+| street               | 123 Tornado Alley           | !!str      | []                          | 19         |
+|                      | Suite 16                    |            |                             |            |
++----------------------+-----------------------------+------------+-----------------------------+------------+
+```
+
+or, you can query configurations of a particular file using:
+
+```sql
+select
+  key_path,
+  value,
+  tag as value_type,
+  pre_comments,
+  start_line
+from
+  yml_key_value
+where
+  path = '/Users/myuser/yml/invoice.yml';
+```
+
+## Examples
```

Comment:
Can you please add 2 examples, one showing `key_path` with `=` and another with `>` or `<` (or the equals versions)?